doc/source/methods/TreeSHAP.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>gurantees --&gt; guarantees</p><p>expasion --&gt; expansion</p><p> </p><p>confused about the notation, we go from h(f(x)) to f(y)=f(h(x)) ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>